Sprint8/Tema4.js

```diff
@@ -0,0 +1,14 @@
+// Returnează numărul de vocale dintr-un șir de caractere (string)
+const arr = ["calculate"]
+function vowelNum(arr) {
```

Comment:
```suggestion
function vowelNum(string) {
```

functia respectiva trebuie sa primeasca un string, nu un array.

De asemenea, recomand sa alegi un nume mai specific al functiei. Un exemplu bun ar putea fi "getVowelsCount"